docs/android/emulator/device-manager.md

```diff
@@ -0,0 +1,426 @@
+---
+title: "Managing Virtual Devices with the Android Device Manager"
+description: "This article explains how to use the Android Device Manager to create and configure Android Virtual Devices (AVDs) that emulate physical Android devices, for .NET MAUI. You can use these virtual devices to run and test your app without having to rely on a physical device"
+ms.date: 11/18/2021
+no-loc: ["user.config"]
+---
+
+# Managing virtual devices with the Android Device Manager
+
+This article explains how to use the Android Device Manager to create and configure Android Virtual Devices (AVDs) that emulate physical Android devices. You can use these virtual devices to run and test your app without having to rely on a physical device.
+
+> [!IMPORTANT]
+> Enable hardware acceleration for the android devices. For more information, see [Hardware Acceleration for Emulator Performance](hardware-acceleration.md).
+
+<!--
+::: zone pivot="windows"
+-->
+
+## Android Device Manager on Windows
+
+You use the Android Device Manager to create and configure an Android Virtual Devices (AVD) that run in the [Android Emulator](debug-on-emulator.md). Each AVD is an emulator configuration that simulates a physical Android device. This makes it possible to run and test your app in a variety of configurations that simulate different physical Android devices.
+
+:::image type="content" source="media/device-manager/win/01-devices-dialog-sml.png" alt-text="Screenshot of the Android Device Manager in the Devices tab." lightbox="media/device-manager/win/01-devices-dialog.png":::
+
+## Requirements
+
+To use the Android Device Manager, you'll need the following items:
+
+- Visual Studio 2022 version 17.1 Preview 3. Visual Studio Community, Professional, and Enterprise editions are supported.
```

Comment:
You can just say Visual Studio 2022 here. No need to specify a version, as it's included in all versions (independent from MAUI).